src/main/java/processbuilder/utils/ProcessBuilderUtilities.java

```diff
@@ -18,7 +18,7 @@
      * @param source       Data source.
      * @param attachedSink Data sink.
      */
-    public void attachToInputStream(final Reader source, final Writer attachedSink) {
+    private void attachToInputStream(final Reader source, final Writer attachedSink) {
         new Thread() {
             public void run() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add the "@Override" annotation above this method signature [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AS1161)